gui/src/app/pages/HomePage/LoadProjectWindow.tsx

```diff
@@ -15,6 +15,9 @@ import {
   useEffect,
   useState,
 } from "react";
+import { Hyperlink, SmallIconButton } from "@fi-sci/misc";
```

Comment:
I'd prefer `@mui.material.Link` over custom `Hyperlink` if possible.

Reply:
switched to <Link> but I needed to style it so the cursor was pointer, for some reason.

Reply:
The [mui docs ](https://mui.com/material-ui/react-link/#accessibility)recommend using `component="button"` if there is no href, which is what the other things use

Reply:
I don't think letting the system style it as a button is appropriate here. By default the text becomes all caps, which is not suitable for titles which are case-sensitive keys.

Reply:
That's not the behavior I observe with e.g. the retry Link in the connection window: https://github.com/flatironinstitute/stan-playground/blob/main/gui/src/app/CompilationServerConnectionControl/CompilationServerConnectionControl.tsx#L137

I believe component="button" doesn't change styling so much as it changes how the browser treats certain events in terms of accessibility and cursor behavior

Reply:
Got it. Made the change.